tests/foreman/cli/test_setting.py

```diff
@@ -451,17 +451,14 @@ def test_positive_failed_login_attempts_limit(setting_update, default_sat):
 
     :BZ: 1778599
     """
-
     username = settings.server.admin_username
     password = settings.server.admin_password
-    result = default_sat.execute(f'hammer -u {username} -p {password} user list')
-    assert result.status == 0
+    assert default_sat.execute(f'hammer -u {username} -p {password} user list').status == 0
     Settings.set({'name': 'failed_login_attempts_limit', 'value': '5'})
     for i in range(5):
```

Comment:
```suggestion
    for _ in range(5):
```
Everything looks good to me. However, common syntax here is to use `_` instead of `i` if you do not use the variable in your loop. Not sure how consistent that is in robottelo, but I like the way it reads :+1: